inst/WORDLIST

```diff
@@ -248,7 +246,6 @@ mmHg
 mmol
 modularized
 msec
-nolint
```

Comment:
Did you intend to modify/remove these entries in `inst/WORDLIST`

Reply:
I wanted to add 'tidyselect' to the WORDLIST and when I did 'spelling::update_wordlist()', these were modified. Should I revert these changes and only add 'tidyselect'?

Reply:
To be on the safe side, I'd say yes but based on the ci/cd checks it didnt seem to affect things, what do you think @bundfussr

Reply:
If the spelling check passes, I see no reason to keep them.